common/v2/services/Asset/types.ts

```diff
@@ -1,14 +1,14 @@
 export interface Asset {
-  option: string;
-  amount: string;
-  network: string;
-  symbol: string;
+  uuid: string;
+  name: string;
+  networkId: string | undefined;
+  ticker: string;
   type: assetMethod;
-  contractAddress?: string;
-  decimal?: number;
+  contractAddress?: string | null;
+  decimal: number | null;
```

Comment:
Could probably be optional instead
```suggestion
  decimal?: number;
```